app/javascript/react/containers/CuisineShowContainer.js

```diff
@@ -52,7 +64,14 @@ class CuisineShowContainer extends Component {
       <div className="colums rows">
         <h1>{this.state.name}</h1>
         {restaurants}
+        <div>
+          <RestaurantFormContainer
+            cuisine_id={this.state.id}
+            updateParent={this.updateRestaurantsList}
```

Comment:
the naming for this prop is too vague

Reply:
yes we do use it, it updated the list of Restaurants in the parent container

Reply:
I changed the name of this prop to make it more precise and readable to updateRestaurantsList